tests/v2/validation/rbac/rbac_psa_test.go

```diff
@@ -376,7 +376,7 @@ func (rb *PSATestSuite) TestPsactRBAC() {
 	for _, tt := range tests {
 
 		rb.Run("Set up User with Cluster Role "+tt.name, func() {
-			newUser, err := createUser(rb.client, tt.member)
+			newUser, err := users.CreateUserWithRole(rb.client, users.UserConfig(), tt.member)
```

Comment:
What is this change doing?

Reply:
I think I messed up the git history. Fixing now